mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1227,6 +1229,12 @@
     <string name="bookmark_deletion_multiple_snackbar_message_3">Deleting selected folders</string>
     <!-- Bookmark undo button for deletion snackbar action -->
     <string name="bookmark_undo_deletion">UNDO</string>
+    <!-- Bookmark snackbar message for deleting a single item. Parameter is the title of the item being deleted -->
+    <string name="bookmark_delete_single_item">Deleted %s</string>
+    <!-- Bookmark snackbar message for deleting multiple items. Parameter is the number of items being deleted -->
+    <string name="bookmark_delete_multiple_items">Deleted %s items</string>
```

Comment:
We still don't handle plural support properly, please update the string to:
Deleted items: %s